tests/unit/testsuite/Bolt/Boltpay/Block/InfoTest.php

```diff
@@ -58,4 +70,38 @@ public function prepareSpecificInformation()
             ], $data->getData()
         );
     }
+    
+    /**
+     * @test
+     * 
+     * @covers ::displayPaymentMethodTitle
+     */
+    public function displayPaymentMethodTitle_ifOrderPaidWithCreditCard_returnDefaultTitle()
+    {
+        $this->currentMock->expects(self::any())->method('getInfo')->willReturn($this->infoMock);
+        $this->currentMock->expects(self::any())->method('getMethod')->willReturn($this->paymentMock);
+        $this->infoMock->expects(self::once())->method('getAdditionalInformation')->with('bolt_processor')
+            ->willReturn('vantiv');
+        $this->paymentMock->expects(self::once())->method('getTitle')->willReturn('Credit and Debit Card (Powered by Bolt)');
```

Comment:
I don't think that the payment getTitle method should be mocked like this.  We should have test for both the frontend area and admin area.  In these cases, values may come from the constants in the admin area (i.e. 'Credit and Debit Card (Powered by Bolt)') or in the frontend area, may come from the admin config which may be set to anything by the merchant.

I will not include the test in the PR that I am submitting and leaving am initially leaving it's implementation up to you.